smtsolver.js

```diff
@@ -56,5 +66,5 @@ function solve (query) {
 
 module.exports = {
   smtSolver: solve,
-  availableSolvers: solvers.length
+  availableSolvers: solvers
```

Comment:
This is still kind of a breaking change, given we export `smtsolver`.

Reply:
and that's ok..? :D

Reply:
@cameel ?

Reply:
This does seem breaking. Unless the whole feature is currently broken enough that no one's really using it. Are people using it in practice?

Maybe we could just add a new member instead?

Reply:
> Unless the whole feature is currently broken enough that no one's really using it

it is completely broken, because the solver passing is broken

Reply:
In that case I think we're fine changing this. It's not like we'll make anyone's code any more broken than it is already :P

Reply:
Oh yeah I remember we discussed to document this in the README and/or Solididty Changelog.

Reply:
Both actually.

Reply:
added to solc-js README